nebula-exchange/src/main/scala/com/vesoft/nebula/exchange/reader/StreamingBaseReader.scala

```diff
@@ -25,17 +28,25 @@ abstract class StreamingBaseReader(override val session: SparkSession) extends R
   * @param session
   * @param kafkaConfig
   */
-class KafkaReader(override val session: SparkSession, kafkaConfig: KafkaSourceConfigEntry)
+class KafkaReader(override val session: SparkSession,
+                  kafkaConfig: KafkaSourceConfigEntry,
+                  fields: List[String])
     extends StreamingBaseReader(session) {
 
-  require(kafkaConfig.server.trim.nonEmpty && kafkaConfig.topic.trim.nonEmpty)
+  require(kafkaConfig.server.trim.nonEmpty && kafkaConfig.topic.trim.nonEmpty && fields.nonEmpty)
 
   override def read(): DataFrame = {
+    import session.implicits._
+    val jsonSchema = StructType(fields.map(field => StructField(field, StringType, true)))
     session.readStream
-      .format("kafka")
-      .option("kafka.bootstrap.servers", kafkaConfig.server)
-      .option("subscribe", kafkaConfig.topic)
-      .load()
+           .format("kafka")
+           .option("kafka.bootstrap.servers", kafkaConfig.server)
+           .option("subscribe", kafkaConfig.topic)
+           .load()
+           .selectExpr("CAST(value AS STRING)")
+           .as[(String)]
+           .withColumn("value", from_json(col("value"), jsonSchema))
+           .select("value.*")
```

Comment:
don't we need alias the dataframe's column name to name in fields?

Reply:
I had ever print the col name which is the name of fields. It's workable in my machine..

Reply:
Yeah, i tested it and the schema is the same with fields. Great work~